x-pack/plugins/fleet/server/services/output.test.ts

```diff
@@ -936,36 +936,7 @@ describe('Output Service', () => {
       const soClient = getMockedSoClient({
```

Comment:
Could we add a similar test with synthetics that verifies the change to default output?

Reply:
More tests added to match fleet server tests.